third_party/packages/split/lib/split.dart

```diff
@@ -57,8 +70,14 @@ class Splitter {
   external void destroy([bool preserveStyles, bool preserveGutters]);
 }
 
-/// Splitter that splits multiple elements that must be styled with flexbox
-/// layout.
+/// Splitter that splits multiple elements using flex layout.
+///
+/// You should used this fixed splitter instead of the fixed splitter if the
```

Comment:
Did you intend for the first "fixed splitter" to be "flex splitter"?

Reply:
done